grpc/client.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package grpc
+
+import (
+	"context"
+	"time"
+
+	chaosv1beta1 "github.com/DataDog/chaos-controller/api/v1beta1"
+	pb "github.com/DataDog/chaos-controller/grpc/disruption_listener"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// ExecuteSendDisruption takes in a CRD specification for GRPC disruptions and
+// executes a SendDisruption call on the provided DisruptionListenerClient
+func ExecuteSendDisruption(client pb.DisruptionListenerClient, spec chaosv1beta1.GRPCDisruptionSpec, logger *zap.SugaredLogger) {
```

Comment:
Personally, I would suggest neither this function nor `ExecuteCleanDisruption` takes a logger. For both functions, they are only doing so in order to log an error. I think it's preferable for them to return an error instead, and have the caller deal with logging